This patch fixes the md plug-in so that it will set the desired parity algorithm
in the superblock for the RAID5 array.  Previously it always set the algorithm
to 0, i.e., left asymmetric.

--- evms-2.5.5/plugins/md/raid5_mgr.c	2006-02-22 10:05:17.000000000 -0600
+++ evms-2.5.5-fix/plugins/md/raid5_mgr.c	2006-06-08 13:59:56.648351368 -0500
@@ -1932,7 +1932,7 @@
 		}
 	}
 
-	rc = md_init_sb(volume, &sb_ver, raid_level, 0, size, chunksize);
+	rc = md_init_sb(volume, &sb_ver, raid_level, parity_algorithm, size, chunksize);
 	if (rc) {
 		goto error_free;
 	}
